packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -213,10 +212,10 @@ test.describe('PieCookieBanner - Component tests', () => {
                 await page.click(getPreferenceItemSelector(preference.id));
             }
         }
-        const isToggleAllChecked = await page.locator(getPreferenceItemSelector('all')).isChecked();
+        const isToggleAllChecked = await page.locator(getPreferenceItemSelector('all'));
```

Comment:
this name suggests a boolean value but it's a locator result which is a little misleading - could you rename it to be clearer please?

Reply:
Fixed.